docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,221 @@
 ---
 title: "Deploy to Kubernetes"
 sidebar_position: 21
----
\ No newline at end of file
+---
+
+TODO - INTRODUCTION
+
+## What you'll learn
+
+- How to use Kubernetes and Helm to deploy Dagster
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Familiarity with [Docker](https://docs.docker.com/)
+- Familiarity with [Kubernetes](https://kubernetes.io/docs/home/)
+- Familiarity with [Helm](https://helm.sh/docs/)
+- A Dagster project to deploy. You can also use the example project [here](/todo). Would be great to have a command they can use to clone the project
```

Comment:
the existing guide has a corresponding example project that you can download. I think it would be nice to do the same thing with the new project, but that could require some additional overhead for keeping the code snippets in this guide up to date with the code in the example project directory. thoughts?

Reply:
it ends up just being the `workspace.yaml` that needs to be maintained in two locations so i think it's worth having a project they can download. I'll make that